src/components/CookieNotice/CookieNotice.tsx

```diff
@@ -0,0 +1,132 @@
+import "./CookieNotice.scss";
+import React from "react";
+import Backdrop from "@material-ui/core/Backdrop";
+import store from "store";
+import {useSelector} from "react-redux";
+import {ApplicationState} from "types/store";
+import {ActionFactory} from "store/action";
+import CookiePolicy from "./CookiePolicy";
+
+/**
+ * Converts string value to boolean value.
+ * @param input string that needs conversion to boolean value.
+ */
+const stringToBoolean = (input: string): boolean => {
+  switch (input.toLowerCase().trim()) {
+    case "true":
+    case "yes":
+    case "1":
+      return true;
+    case "false":
+    case "no":
+    case "0":
+    case null:
+      return false;
+    default:
+      return false;
+  }
+};
+
+const CookieNotice = () => {
+  /** cookie name */
+  const scrumlrCookieName = "scrumlr_cookieConsent";
+
+  const cookieState = useSelector((applicationState: ApplicationState) => ({
+    cookieConsent: applicationState.cookieConsent,
+  }));
+
+  if (localStorage.getItem(scrumlrCookieName)) {
+    cookieState.cookieConsent.name = scrumlrCookieName;
+    cookieState.cookieConsent.value = stringToBoolean(localStorage[scrumlrCookieName]);
+  }
+
+  const [showCookieNotice, setShowCookieNotice] = React.useState(true); // state: states whether cookie notice is shown.
+  const [open, setOpen] = React.useState(true); // state: states whether backdrop is open.
+
+  const [showDialog, setShowDialog] = React.useState(false);
+  const handleShowDialog = () => {
```

Comment:
I would name this function `toggleShowDialog` to make it more explicit. Might be just me, but `handle` implies that i probably can pass an argument